spec/requests/api/v1/topic/topics_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe '/topics', type: :request do
+  subject { get api_v1_topics_url }
+  let!(:topics) { create_list(:topic, 3) }
+
+  describe 'GET /index' do
+    it 'how many topics' do
```

Comment:
`it 'how many topics' do`  sounds wierd right? The `it` description should be `it does something`
Maybe we can use `it 'returns all topcis' do`

Reply:
Also, are we testing that response has all topics attributes we want ?

Reply:
add the description 
`it 'renders a successful response' do`